docs/fluo-recipes/1.0.0-incubating/transient.md

```diff
@@ -25,7 +25,7 @@ standard way.
 
 ## Registering Transient Ranges
 
-Recipes like [Export Queue](export-queue.md) will automatically register
+Recipes like [Export Queue](../export-queue/) will automatically register
```

Comment:
How did you find these links were broken?  Was it a build error?

Reply:
Whenever I update / change our dependencies for the static build of the website, I always do a recursive diff of the entire built `_site/` directory to see how it affects the generated site. These ones showed up as part of that. It seems that some gem that was removed was doing some additional translation of these links to their generated equivalent, but it stopped working after dropping those gems. Since there was only a few like this, it was an easy fix to just update the link manually rather try to track down which gem was responsible for that magic and re-add it.